Jenkinsfile

```diff
@@ -146,16 +164,15 @@ pipeline {
       steps {
         script {
           lastStage = env.STAGE_NAME
-          // uses env.DOCKER_TAG and deploys to staging by default
-          marathonDeploy(serviceName: env.DEPLOY_PATTERN)
+          marathonDeploy(serviceName: env.DEPLOY_PATTERN, tag: env.DOCKER_TAG, environment: 'staging')
         }
       }
     }
   }
   post {
     failure {
       script {
-        if (!isPr) {
+        if (env.JOB_NAME == "${service}/main") {
```

Comment:
Did isPR stop working?

Reply:
No, but it's not specific enough.  When I am testing on a branch, I don't want the build failures to post to teams.  We only want the main build failures to post to teams.  Any release build failures will show up in the "Release-Build" job, so we'll see them there.

Reply:
For clarity, I'd express as such:
```groovy
boolean buildingMain = env.JOB_NAME = "${service}/main"
if (buildingMain) {...}
```

Reply:
Hmm, I'll think about this suggestion for future PRs.